public/js/icinga/ui.js

```diff
@@ -701,6 +704,43 @@
             }
         },
 
+        ensureTabVisibility: function ($controls) {
+            var $tabContainer = $controls.find('.tabs', $controls);
+            var $tabs = $tabContainer.children('li');
+            var $dropdown = $tabContainer.find('.tabs-dropdown');
+            var $visibleCount = $tabs.has(':visible').size();
```

Comment:
`$tabs.filter(':visible')` has better performance as per the [documentation](https://api.jquery.com/visible-selector/). And since we're only interested in the tabs themselves (not their content) this suffices.